lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -254,6 +305,70 @@ describe('BasicLtiLaunchApp', () => {
     });
   });
 
+  describe('multiple fetches', () => {
+    let contentUrlCall;
+    let groupsCall;
+
+    beforeEach(() => {
+      // Will attempt to fetch the 1. content url and 2. groups.
+      fakeConfig.api = {
+        authToken: 'dummyAuthToken',
+        viaCallbackUrl: 'https://lms.hypothes.is/api/files/1234',
+        sync: {
+          data: {
+            course: {
+              context_id: '12345',
+              custom_canvas_course_id: '101',
+            },
+          },
+          path: '/api/sync',
+        },
+      };
+      contentUrlCall = fakeApiCall.withArgs({
+        authToken: 'dummyAuthToken',
+        path: 'https://lms.hypothes.is/api/files/1234',
+      });
+
+      groupsCall = fakeApiCall.withArgs({
+        authToken: 'dummyAuthToken',
+        path: '/api/sync',
+        data: {
+          course: {
+            context_id: '12345',
+            custom_canvas_course_id: '101',
+          },
+        },
+      });
+    });
+    it('renders the spinner until both groups and contentUrl requests finish', async () => {
```

Comment:
Nit. Can you add a blank line between tests for readability please? (Just the tests in this block are affected)

Reply:
I will, I missed those.